umbraco/Umbraco.CMS.Backoffice
# Feature: Route not found + router features

Implementing Not Found views for our routers.

This includes making the router-slot more reactive, responding to late appearing Routes. As well making sure to cancel a Route Resolving if a new Route appears in the mean time. (its clear now, that the Router Slots routes was never made to be changing over time, but now I have made it so. )

This also fixes a problem when switching between variants, when returning to a variant that was opened earlier, you ended at a blank screen before. (Which was repported in issue as well, found it after this PR was made: ( Fixes https://github.com/umbraco/Umbraco-CMS/issues/16460 )

Be aware this affects almost the whole application, so please try to come up with some edge case scenario and test that.

Fixes https://github.com/umbraco/Umbraco-CMS/issues/16382

## Types of changes




## Motivation and context



## How to test?

Type wrong URLs and see the 'Not Found' page appearing.

As well testing late appearing and later removed Routes, you can add this condition to a extension of choice that provides a Route:
```
{
				alias: 'Umb.Condition.Switch',
				frequency: '2000'
			}
```

## Screenshots (if appropriate)

## Checklist
